### PR TITLE
better error message when git is not installed

### PR DESCRIPTION
**Pull request by dsokal**
# Checklist

- [x] I've added an entry to [CHANGELOG.md](https://github.com/expo/eas-cli/blob/main/CHANGELOG.md#main) if necessary. You can comment this pull request with `/changelog-entry [breaking-change|new-feature|bug-fix|chore] [message]` and CHANGELOG.md will be updated automatically.

# Why

Closes https://github.com/expo/eas-cli/issues/779

# How

Make the error message more descriptive.

# Test Plan

Screenshot 2021-11-23 at 16 42 18

### CODE REVIEW

**Comment by dsokal** [comment]
/changelog-entry chore Make missing `git` command error message more descriptive.

File: packages/eas-cli/src/vcs/clients/git.ts [hunk 1]
```diff
@@ -16,7 +17,13 @@ import { Client } from '../vcs';
 export default class GitClient extends Client {
   public async ensureRepoExistsAsync(): Promise<void> {
     if (!(await isGitInstalledAsync())) {
-      throw new Error('git command not found, install it before proceeding');
+      Log.error(
+        `${chalk.bold('git')} command not found. Install it before proceeding or set ${chalk.bold(
+          'EAS_NO_VCS=1'
+        )} if you wish to use EAS CLI without Git.`
```
Comment (brentvatne) [inline packages/eas-cli/src/vcs/clients/git.ts:23]:
```suggestion
        )} to use EAS CLI without Git.`
```

**Review by brentvatne — APPROVED** [review]

File: packages/eas-cli/src/vcs/clients/git.ts [hunk 2]
```diff
@@ -16,7 +17,13 @@ import { Client } from '../vcs';
 export default class GitClient extends Client {
   public async ensureRepoExistsAsync(): Promise<void> {
     if (!(await isGitInstalledAsync())) {
-      throw new Error('git command not found, install it before proceeding');
+      Log.error(
+        `${chalk.bold('git')} command not found. Install it before proceeding or set ${chalk.bold(
+          'EAS_NO_VCS=1'
+        )} to use EAS CLI without Git.`
```
Comment (wkozyra95) [inline packages/eas-cli/src/vcs/clients/git.ts:23]: `without any version control systems` or `without Git or any other version control systems` ?

**Review by wkozyra95 — APPROVED** [review]

File: packages/eas-cli/src/vcs/clients/git.ts (hunk 2 above)
Comment (dsokal) [reply packages/eas-cli/src/vcs/clients/git.ts:23]:
```suggestion
        )} to use EAS CLI without Git (or any other version control system).`
```

**Pull request merged**